DigitalLearningSolutions.Data.Migrations/Scripts/UAR-831-CreateViewsForAdminUsersAndCandidatesTables-UP.sql

```diff
@@ -0,0 +1,80 @@
+﻿CREATE VIEW AdminUsers AS

```

Comment:
We shouldn't have any deprecated columns in here. These views are going to be used by live queries, so if they're pulling in deprecated data we'll get bugs.
We should either use new columns or hard-code values.

Reply:
Hard-code null or empty values for the columns we're removing.

Reply:
done